Apps/phjira/jira.json

```diff
@@ -5306,6 +5729,240 @@
             ],
             "versions": "EQ(*)"
         },
+        {
+            "action": "lookup users",
+            "description": "Get a list of user resources that match the specified search string",
+            "type": "investigate",
+            "identifier": "search_users",
+            "read_only": true,
+            "verbose": "This action will be used to fetch username of user resources for Jira on-prem and account_id of user resources for Jira cloud. The default value for [max_results] action parameter is <b>1000</b>. The maximum number of tickets as specified by the parameter [max_results] will be fetched starting from the first.<h3>Caveats</h3>Jira Cloud is removing the username field from user profiles in Atlassian Cloud sites. They are also removing username support from their product API's for Jira Cloud. Since is not possible to search users using username for Jira cloud, we will use user's display name to search users. You can use [display_name] action parameter to search users for Jira cloud, and, [username] action parameter will be use to search users for Jira on-prem.",
```

Comment:
The maximum number of **tickets** as specified by the parameter.

"tickets" should be "users"

Reply:
Fixed